exercises/isogram/.meta/solutions/isogram.rb

```diff
@@ -1,5 +1,5 @@
 module BookKeeping
-  VERSION = 4
+  VERSION = 5
```

Comment:
I don't believe this changes the tests functionally, and would not break any solution that is in place, so the version will not need to be bumped.